garden-backend-service/src/models/benchmark.py

```diff
@@ -50,21 +50,21 @@ class BenchmarkRun(Base):
     )
 
     task_id: Mapped[int] = mapped_column(
```

Comment:
I believe we'll also need the annotation to read `Mapped[int | None]` and/or pass `nullable=True` kwarg to the mapped_column (like the regular `ModalInvocationLog/Result` models), otherwise iirc the cascade will try to set null but fail since the column isn't nullable